### DISCUSSION TITLE
Skipping TLS verification

### DISCUSSION THREAD

**Discussion by baibhav73** [vmware-tanzu/velero#5153] [Community support Q&A]
I have installed velero using helm, it automatically uses bucket.host.domain/object for which I am getting certificate is invalid error, in our environment certificate is valid for host.domain/bucket/object, not bucket.host.domain/object 

time="2022-07-13T19:50:11Z" level=error msg="Error listing backups in backup store" backupLocation=default controller=backup-sync error="rpc error: code = Unknown desc = RequestError: send request failed\ncaused by: Get \"[https://bucket.host.domain/object 
) x509: certificate is valid for host.domain/bucket/object, notbucket.host.domain/object " error.file="/go/src/velero-plugin-for-aws/velero-plugin-for-aws/object_store.go:368" error.function="main.(*ObjectStore).ListCommonPrefixes" logSource="pkg/controller/backup_sync_controller.go:182"

I have tried using [this](https://velero.io/docs/v1.9/self-signed-certificates/#skipping-tls-verification) for Skipping TLS verification   by setting Spec.Config.InsecureSkipTLSVerify to true in the [BackupStorageLocation](https://velero.io/docs/v1.9/api-types/backupstoragelocation/) CRD, but CRD seems to revert --insecure-skip-tls-verif

**Comment by VickyWinner** [comment]
@baibhav73 Were you able to fix this issue? Mind sharing your resolution pls?

**Reply by baibhav73** [reply]
Not yet, still waiting for our internal team to whitelist the certificate
